Assignment8/Tracking Application/Tracking Application/MainWindow.xaml.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Timers;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using System.Windows.Threading;
+
+namespace TrackingApplication
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        private const string StartTimer = "0:00:00";
+        public Timer Timer { get; set; }
+
+        public List<(string Timer, string Description)> Records { get; private set; }
```

Comment:
Can be made readonly.
```suggestion
        public List<(string Timer, string Description)> Records { get; }
```